### PR TITLE
Reformat and normalise population table

### PR DESCRIPTION
**Pull request by rafa-garcia**
These are enhancements to the data table rather than bugs

### Workflow

 - [x] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-1093 
 - [x] Jira ticket has the correct status.
 - [x] [Changelog](CHANGELOG.md) entry added.
 
### Merging

- [ ] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: react-components/src/components/CompareMarkets/CompareMarkets.test.jsx [hunk 1]
```diff
@@ -108,10 +108,10 @@ it('Allows selection of markets and fetch data when product selected', async ()
...
   const rowGermany = container.querySelector('#market-Germany')
   expect(rowGermany.querySelector('.name').textContent).toMatch('Germany')
-  expect(rowGermany.querySelector('.total-population').textContent).toMatch('60.46 million')
-  expect(rowGermany.querySelector('.internet-usage').textContent).toMatch('74.39%')
-  expect(rowGermany.querySelector('.urban-population').textContent).toMatch('69.48% (42.01 million)')
-  expect(rowGermany.querySelector('.rural-population').textContent).toMatch('28.32% (17.12 million)')
+  expect(rowGermany.querySelector('.total-population').textContent).toMatch('60.5 million')
+  expect(rowGermany.querySelector('.internet-usage').textContent).toMatch('74%')
+  expect(rowGermany.querySelector('.urban-population').textContent).toMatch('70% 42 million')
+  expect(rowGermany.querySelector('.rural-population').textContent).toMatch('28% 17.1 million')
```
Comment (webbyfox) [inline react-components/src/components/CompareMarkets/CompareMarkets.test.jsx:114]: Just glanced at this, we still required parenthesis around those millions, no?

File: core/sass/learn/_market-finder.scss [hunk 2]
```diff
@@ -9,7 +9,8 @@
   }
   table td,
   table th {
-    border-bottom: 1px solid $blue-deep-20;
+    text-align: right !important;
+    border-bottom: 1px solid $blue-deep-20 !important;
```
Comment (webbyfox) [inline core/sass/learn/_market-finder.scss:13]: FE might like this `!important`

File: react-components/src/components/CompareMarkets/CompareMarkets.test.jsx (hunk 1 above)
Comment (rafa-garcia) [reply react-components/src/components/CompareMarkets/CompareMarkets.test.jsx:114]: No, the count now shows in a new line with no parentheses

File: core/sass/learn/_market-finder.scss (hunk 2 above)
Comment (rafa-garcia) [reply core/sass/learn/_market-finder.scss:13]: It won't pick it up otherwise. Suggestions?

**Review by webbyfox — APPROVED** [review]
LGTM

**Pull request merged**